File: src/main/java/models/feriel/Testimony.java
package models.feriel;

public class Testimony {
    int t_id;
    String title;
    String txt;
    int userId;

    String status;


    public Testimony(  int userId,String title, String txt) {

        this.title = title;
        this.txt = txt;
        this.userId = userId;
        this.status= "processing";
    }
    public Testimony( ) {}
    public int getT_id() {
        return t_id;
    }

    @Override
    public String toString() {
        return "Testimony{" +
                "id=" + t_id +
                ", title='" + title + '\'' +
                ", txt='" + txt + '\'' +
                ", userId=" + userId +
                '}';
    }

    public void setT_id(int t_id) {
        this.t_id = t_id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getTxt() {
        return txt;
    }

    public void setTxt(String txt) {
        this.txt = txt;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }
    public String getStatus() {return status;}

    public void setStatus(String status) {this.status = status;}




}

File: src/Entity/Testimony.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Testimony
 *
 * @ORM\Table(name="testimony")
 * @ORM\Entity
 */
class Testimony
{
    /**
     * @var int
     *
     * @ORM\Column(name="t_id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $tId;

    /**
     * @var int
     *
     * @ORM\Column(name="userId", type="integer", nullable=false)
     */
    private $userid;

    /**
     * @var string
     *
     * @ORM\Column(name="title", type="string", length=255, nullable=false)
     */
    private $title;

    /**
     * @var string
     *
     * @ORM\Column(name="txt", type="string", length=1000, nullable=false)
     */
    private $txt;

    /**
     * @var string|null
     *
     * @ORM\Column(name="status", type="string", length=50, nullable=true)
     */
    private $status;

    public function getTId(): ?int
    {
        return $this->tId;
    }

    public function getUserid(): ?int
    {
        return $this->userid;
    }

    public function setUserid(int $userid): static
    {
        $this->userid = $userid;

        return $this;
    }

    public function getTitle(): ?string
    {
        return $this->title;
    }

    public function setTitle(string $title): static
    {
        $this->title = $title;

        return $this;
    }

    public function getTxt(): ?string
    {
        return $this->txt;
    }

    public function setTxt(string $txt): static
    {
        $this->txt = $txt;

        return $this;
    }

    public function getStatus(): ?string
    {
        return $this->status;
    }

    public function setStatus(?string $status): static
    {
        $this->status = $status;

        return $this;
    }


}
